client/src/components/UI/Input.jsx

```diff
@@ -35,6 +35,17 @@ const Input = props => {
                 />
             );
             break;
+        case 'file':
+            element = (
+                <Form.Input
+                    onChange={event =>
+                        props.onChange(event, { value: event.target.files[0] })
+                    }
+                    error={props.error}
+                    {...props.elementConfig}
+                />
+            );
+            break;
         default:
```

Comment:
can u just define it before cases and get rid of default

Reply:
linters insist on having default case in switch

Reply:
ok